python/ray/util/client/server/dataservicer.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, basic_service: "RayletServicer"):
 
     def Datapath(self, request_iterator, context):
         metadata = {k: v for k, v in context.invocation_metadata()}
-        client_id = metadata["client_id"]
+        client_id = metadata.get("client_id") or ""
```

Comment:
`client_id = metadata.get("client_id", "")`

Reply:
I want to ensure that it is a string. The problem with `metadata.get("client_id", "")` is that if `metadata  = {"client_id" : None}`, client_id will be `None`, not `""`.